eth2near/contract_wrapper/src/eth_client_contract.rs

```diff
@@ -60,7 +60,7 @@ impl EthClientContract {
             verify_bls_signatures: false,
             hashes_gc_threshold: 51000,
             max_submitted_blocks_by_account: 8000,
-            trusted_signer: Option::<AccountId>::None,
+            trusted_signer: Option::<AccountId>::Some(self.contract_wrapper.get_signer_account_id()),
```

Comment:
Let's utilize the approach in #819 after that one is merged.

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/818/commits/4e20075373f66594873fdb5a86b0ad67dadaa62a